configurations/dadaab_pipeline_config.json

```diff
@@ -0,0 +1,81 @@
+{
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/wusc-keep-II-dadaab.textit-token.txt",
+      "ContactsFileName": "wusc_keep_II",
+      "ActivationFlowNames": [
+        "keep_II_s01_e01_dadaab_activation",
+        "keep_II_s01_e02_dadaab_activation",
+        "keep_II_s01_e03_dadaab_activation",
+        "keep_II_s01_e04_dadaab_activation",
+        "keep_II_s01_e05_dadaab_activation",
+        "keep_II_s01_e06_dadaab_activation",
+        "keep_II_s01_e07_dadaab_activation"
+      ],
+      "SurveyFlowNames": [
+        "keep_II_dadaab_demogs"
+      ],
+      "TestContactUUIDs": [
+        "f3e07a4a-4c5d-4032-80f3-c31bcf480dc9"
+      ]
+    }
+  ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "WUSC_phone_number_avf_phone_id"
+  },
+  "RapidProKeyRemappings": [
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
+    {"RapidProKey": "Rqa_S01_E01_Dadaab (Value) - keep_II_s01_e01_dadaab_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
```

Comment:
Don't think we need to change now, but if we ever combine datasets it might make sense to include the location in the mapped key name e.g. "rqa_dadaab_s01e01_raw"